src/build.d

```diff
@@ -820,6 +821,29 @@ auto detectModel()
     throw new Exception(`Cannot figure 32/64 model from "` ~ uname ~ `"`);
 }
 
+/*
+Gets the command for querying or invoking the host C++ compiler
+
+Returns: the command for querying or invoking the host C++ compiler
+*/
+auto getHostCXX()
+{
+    version(Posix)
+        return "c++";
+    else version(Windows)
+    {
+        immutable model = detectModel;
+        if (model == "32")
+            return "dmc";
+        else if (model == "64")
+            return `vcbuild\msvc-dmc`;
```

Comment:
See https://github.com/dlang/dmd/blob/fd66f64bf8f8720b5248a8c3f836c899d55e4676/src/win64.mak#L24

`msvc-dmc` can be found in https://github.com/dlang/dmd/tree/master/src/vcbuild

Reply:
My understanding is that for 32-bit code, the makefiles us DMC:  https://github.com/dlang/dmd/blob/fd66f64bf8f8720b5248a8c3f836c899d55e4676/src/win32.mak#L88

Reply:
I think you need to make sure that this file is compiled.

```d
Dependency dependency = {
        target: env["G"].buildPath("msvc-dmc").exeName,
        sources: ["vcbuild\msvc-dmc"],
};
dependency.run();
```

(though if you put this in `$G`, make sure `mkdirRecurse(g);` (also part of `parseEnvironment` is called first)).

Reply:
If you're implying that we should be using MSVC for both 32 and 64-bit code, well, I'm trying not to rock the boat.

Reply:
> I think you need to make sure that this file is compiled.

Yes, I'm just working through the script step by step.  I'll get there.

Reply:
And looking a bit more at the win64.mak file, it seems like it also requires to build the `vcbuild\msvc-lib` file

Reply:
Added `msvc-lib`, but they're not building yet, that's a bigger task than I intended to tackle today.

Reply:
Okay, but I think you still need sth. like `buildMsvcDmc().target` here to return the correct path.